src/test/java/core/basesyntax/service/impl/OperationStrategyImplTest.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.OperationHandler;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+class OperationStrategyImplTest {
+
+    @Test
+    public void getHandler_balanceOperation() {
+        Map<FruitTransaction.Operation, OperationHandler> operationHandlers = new HashMap<>();
+        operationHandlers.put(FruitTransaction.Operation.BALANCE, new BalanceOperation());
+        OperationStrategyImpl strategy = new OperationStrategyImpl(operationHandlers);
```

Comment:
why not be consistent and create the class under the test in a separate beforeall / beforeeach method?